### PR TITLE
mv Ricardo slides with his presentation file

### PR DESCRIPTION
**Pull request by reshamas**
Ricardo's slides pdf file is in the static images folder.
This PR moves it to its folder in about/contributing_to_pymc which is consistent with the other resources. 

Current folder:
https://pymc-data-umbrella.xyz/en/latest/_static/contributing_to_pymc_slides.pdf

New folder:
https://pymc-data-umbrella.xyz/en/latest/about/contributing_to_pymc/contributing_to_pymc_slides.pdf

:books: Documentation preview pt :books:: https://pymc-data-umbrella--202.org.readthedocs.build/pt/202/

### CODE REVIEW

**Comment by reshamas** [comment]
@OriolAbril 
The link is broken, though I am not sure where the error is.  It looks like I put the correct path in.

File: about/contributing_to_pymc/index.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ From first time contributor to project maintainer. What is it like? In this test
 
 - Audience: People who are interested in contributing to Open source
 - [Github Repo](https://github.com/pymc-devs/pymc-data-umbrella)
-- [Slides](https://pymc-data-umbrella.xyz/en/latest/_static/contributing_to_pymc_slides.pdf)
+- [Slides](https://pymc-data-umbrella.xyz/en/latest/about/contributing_to_pymc/contributing_to_pymc_slides.pdf)
```
[the remark this answers is not in the document]
Comment (OriolAbril) [reply about/contributing_to_pymc/index.md]: I don't know what kind of link is generated by the download role, we'll see

[the remark this answers is not in the document]
Comment (OriolAbril) [reply about/contributing_to_pymc/index.md]: If this forces the download we'll have to go back to the original version, and probably add a comment to clarify what is happening

File: about/contributing_to_pymc/index.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ From first time contributor to project maintainer. What is it like? In this test
 
 - Audience: People who are interested in contributing to Open source
 - [Github Repo](https://github.com/pymc-devs/pymc-data-umbrella)
-- [Slides](https://pymc-data-umbrella.xyz/en/latest/_static/contributing_to_pymc_slides.pdf)
+- {download}`Slides <contributing_to_pymc_slides.pdf>`
```
Comment (reshamas) [inline about/contributing_to_pymc/index.md:11]:
```suggestion
- {ref}`Slides <contributing_to_pymc_slides.pdf>`
```

File: about/contributing_to_pymc/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (reshamas) [reply about/contributing_to_pymc/index.md]: Using `ref` doesn't seem to work.

File: about/contributing_to_pymc/index.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ From first time contributor to project maintainer. What is it like? In this test
 
 - Audience: People who are interested in contributing to Open source
 - [Github Repo](https://github.com/pymc-devs/pymc-data-umbrella)
-- [Slides](https://pymc-data-umbrella.xyz/en/latest/_static/contributing_to_pymc_slides.pdf)
+- {ref}`Slides <contributing_to_pymc_slides.pdf>`
```
Comment (reshamas) [inline about/contributing_to_pymc/index.md:11]:
```suggestion
- [Slides](contributing_to_pymc_slides.pdf)
```

**Comment by OriolAbril** [comment]
I don't think it is necessary to actually download the file. The download role generates a link to https://pymc-data-umbrella--202.org.readthedocs.build/en/202/_downloads/f8020df2856116faddd0ce9a59c303f1/contributing_to_pymc_slides.pdf, so the effect is basically the same as we had originally.

The `ref` doesn't work because it is to reference manually set targets (e.g. using the `(target)=` syntax. The markdown link acts as a download one because it is kind of a wildcard, with myst choosing the sphinx role that better suits (or at least what myst thinks it suits better). It doesn't add any extra functionality though

File: about/contributing_to_pymc/index.md [hunk 4]
```diff
@@ -8,7 +8,7 @@ From first time contributor to project maintainer. What is it like? In this test
 
 - Audience: People who are interested in contributing to Open source
 - [Github Repo](https://github.com/pymc-devs/pymc-data-umbrella)
-- [Slides](https://pymc-data-umbrella.xyz/en/latest/_static/contributing_to_pymc_slides.pdf)
+- [Slides](contributing_to_pymc_slides.pdf)
```
Comment (reshamas) [inline about/contributing_to_pymc/index.md:11]:
```suggestion
{download}`Slides <contributing_to_pymc_slides.pdf>`
```

File: locales/es/LC_MESSAGES/about/contributing_to_pymc/index.po [hunk 5]
```diff
@@ -62,10 +62,10 @@ msgstr "[Repositorio Github](https://github.com/pymc-devs/pymc-data-umbrella)"
 #: ../about/contributing_to_pymc/index.md:11 a367060ca11c467095de8d3aef2bac4e
 msgid ""
 "[Slides](https://pymc-data-"
-"umbrella.xyz/en/latest/_static/contributing_to_pymc_slides.pdf)"
+"umbrella.xyz/en/latest/about/contributing_to_pymc/contributing_to_pymc_slides.pdf)"
```
Comment (OriolAbril) [inline locales/es/LC_MESSAGES/about/contributing_to_pymc/index.po:65]:
```suggestion
"umbrella.xyz/en/latest/_static/contributing_to_pymc_slides.pdf)"
```

File: locales/es/LC_MESSAGES/about/contributing_to_pymc/index.po [hunk 6]
```diff
@@ -62,10 +62,10 @@ msgstr "[Repositorio Github](https://github.com/pymc-devs/pymc-data-umbrella)"
...
 msgstr ""
 "[Diapositivas](https://pymc-data-"
-"umbrella.xyz/en/latest/_static/contributing_to_pymc_slides.pdf)"
+"umbrella.xyz/en/latest/about/contributing_to_pymc/contributing_to_pymc_slides.pdf)"
```
Comment (OriolAbril) [inline locales/es/LC_MESSAGES/about/contributing_to_pymc/index.po:68]:
```suggestion
"umbrella.xyz/en/latest/_static/contributing_to_pymc_slides.pdf)"
```

File: locales/pt/LC_MESSAGES/about/contributing_to_pymc/index.po [hunk 7]
```diff
@@ -63,10 +63,10 @@ msgstr "[Repo no Github](https://github.com/pymc-devs/pymc-data-umbrella)"
 #: ../about/contributing_to_pymc/index.md:11 a367060ca11c467095de8d3aef2bac4e
 msgid ""
 "[Slides](https://pymc-data-"
-"umbrella.xyz/en/latest/_static/contributing_to_pymc_slides.pdf)"
+"umbrella.xyz/en/latest/about/contributing_to_pymc/contributing_to_pymc_slides.pdf)"
```
Comment (OriolAbril) [inline locales/pt/LC_MESSAGES/about/contributing_to_pymc/index.po:66]:
```suggestion
"umbrella.xyz/en/latest/_static/contributing_to_pymc_slides.pdf)"
```

File: locales/pt/LC_MESSAGES/about/contributing_to_pymc/index.po [hunk 8]
```diff
@@ -63,10 +63,10 @@ msgstr "[Repo no Github](https://github.com/pymc-devs/pymc-data-umbrella)"
...
 msgstr ""
 "[Apresentação](https://pymc-data-"
-"umbrella.xyz/en/latest/_static/contributing_to_pymc_slides.pdf)"
+"umbrella.xyz/en/latest/about/contributing_to_pymc/contributing_to_pymc_slides.pdf)"
```
Comment (OriolAbril) [inline locales/pt/LC_MESSAGES/about/contributing_to_pymc/index.po:69]:
```suggestion
"umbrella.xyz/en/latest/_static/contributing_to_pymc_slides.pdf)"
```

**Review by OriolAbril — APPROVED** [review]
I have undone the changes to the .po files.

While it is possible to edit the po files both locally and on transifex, our current CI config assumes they are only modified on transifex. I think if we do changes to po files locally these changes will be overwritten whenever we pull from transifex, which isn't super bad but not ideal. However, worst case scenario it will create git conflicts and break the auto PR action

**Pull request merged**